Installing on Mac

Hi

great job, I'm using it on windows, but would like to use it on Mac as well.

I did follow the install guide, but seems not to work for me : 

`toto@Mac-mini ~ % cd Chia-Plot-Status
toto@Mac-mini Chia-Plot-Status % dotnet build --configuration .\ChiaPlotStatus.sln /p:Configuration=Release /p:Platform="Any CPU"

Bienvenue dans .NET 5.0 !
---------------------
Version du kit SDK : 5.0.202

Télémétrie
---------
Les outils .NET collectent des données d'utilisation qui nous aident à améliorer votre expérience utilisateur. Elles sont collectées par Microsoft et partagées par la communauté. Vous pouvez refuser l'adhésion à la télémétrie en affectant la valeur '1' ou 'true' à la variable d'environnement DOTNET_CLI_TELEMETRY_OPTOUT via l'interpréteur de commandes de votre choix.

Pour plus d'informations sur la télémétrie des outils CLI .NET, accédez à https://aka.ms/dotnet-cli-telemetry

----------------
Un certificat de développement HTTPS ASP.NET Core a été installé.
Pour approuver le certificat, exécutez 'dotnet dev-certs https --trust' (Windows et macOS uniquement).
Découvrez HTTPS : https://aka.ms/dotnet-https
----------------
Écrivez votre première application : https://aka.ms/dotnet-hello-world
Découvrez les nouveautés : https://aka.ms/dotnet-whats-new
Consultez la documentation : https://aka.ms/dotnet-docs
Signalez des problèmes et recherchez du code source sur GitHub : https://github.com/dotnet/core
Utilisez 'dotnet --help' pour voir les commandes disponibles, ou accédez à https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------
Microsoft (R) Build Engine version 16.9.0+57a23d249 pour .NET
Copyright (C) Microsoft Corporation. Tous droits réservés.

  Identification des projets à restaurer...
  Restauration effectuée de /Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj (en 24,74 sec).
  GenerateAvaloniaResourcesTask -> Root: /Users/toto/Chia-Plot-Status, 11 resources, Output:obj/Release/net5.0//Avalonia/resources
  avares -> name:/Assets/avalonia-logo.ico, path: /Users/toto/Chia-Plot-Status/Assets/avalonia-logo.ico, size:176111, ItemSpec:Assets/avalonia-logo.ico
  avares -> name:/Assets/en.yaml, path: /Users/toto/Chia-Plot-Status/Assets/en.yaml, size:9353, ItemSpec:Assets/en.yaml
  avares -> name:/Assets/Logo.jpg, path: /Users/toto/Chia-Plot-Status/Assets/Logo.jpg, size:5879, ItemSpec:Assets/Logo.jpg
  avares -> name:/Assets/Logo.png, path: /Users/toto/Chia-Plot-Status/Assets/Logo.png, size:4855, ItemSpec:Assets/Logo.png
  avares -> name:/Assets/version.yaml, path: /Users/toto/Chia-Plot-Status/Assets/version.yaml, size:26, ItemSpec:Assets/version.yaml
  avares -> name:/GUI/App.axaml, path: /Users/toto/Chia-Plot-Status/GUI/App.axaml, size:823, ItemSpec:GUI/App.axaml
  avares -> name:/GUI/Views/DonationDialog.axaml, path: /Users/toto/Chia-Plot-Status/GUI/Views/DonationDialog.axaml, size:2844, ItemSpec:GUI/Views/DonationDialog.axaml
  avares -> name:/GUI/Views/LogViewerWindow.axaml, path: /Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml, size:2138, ItemSpec:GUI/Views/LogViewerWindow.axaml
  avares -> name:/GUI/Views/MainWindow.axaml, path: /Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml, size:33458, ItemSpec:GUI/Views/MainWindow.axaml
  avares -> name:/GUI/Views/MarkOfDeathDialog.axaml, path: /Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml, size:1886, ItemSpec:GUI/Views/MarkOfDeathDialog.axaml
  avares -> name:/GUI/Views/UpdateDialog.axaml, path: /Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml, size:1810, ItemSpec:GUI/Views/UpdateDialog.axaml
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'Language' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'Latest' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'Current' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'current' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'PlotManager' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'ExportJsonCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'ExportYamlCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'ExportCsvCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'AddFolderCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'RemoveFolderCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'IncreaseFontSizeCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'DecreaseFontSizeCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'MarkAsDeadCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/DonationDialog.axaml.cs(26,16): warning CS8618: Le propriété 'Language' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(35,16): warning CS8618: Le propriété 'current' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/DonationDialog.axaml.cs(47,27): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/DonationDialog.axaml.cs(48,56): warning CS8604: Existence possible d'un argument de référence null pour le paramètre 'text' dans 'Task IClipboard.SetTextAsync(string text)'. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le propriété 'TailLineEmitter' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le propriété 'Timer' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le propriété 'Path' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le champ 'LogLines' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le champ comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(32,16): warning CS8618: Le propriété 'TailLineEmitter' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(32,16): warning CS8618: Le propriété 'Timer' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(32,16): warning CS8618: Le champ 'LogLines' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le champ comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'BtnClickWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'TextChangeWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'SortChangeWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'ThemeSwitchWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(52,27): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(26,25): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(27,25): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(31,25): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(23,16): warning CS8618: Le propriété 'Tmp1Drive' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(23,16): warning CS8618: Le propriété 'Tmp2Drive' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(70,13): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(98,75): warning CS8629: Le type valeur Nullable peut avoir une valeur null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(103,58): warning CS8604: Existence possible d'un argument de référence null pour le paramètre 'arg' dans 'bool Func<string, bool>.Invoke(string arg)'. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(66,20): warning CS8603: Existence possible d'un retour de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(36,23): warning CS8618: Le propriété 'Name' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(37,25): warning CS8618: Le propriété 'Tooltips' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(38,24): warning CS8618: Le propriété 'Columns' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(39,23): warning CS8618: Le propriété 'Fields' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(40,24): warning CS8618: Le propriété 'Buttons' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(91,35): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(92,24): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'Settings' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'Language' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'OnUpdate' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'plotLogReadable' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(43,41): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(58,18): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(70,17): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(109,51): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/ChiaPlotStatus.cs(67,45): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/MarkOfDeath.cs(19,16): warning CS8618: Le propriété 'LogFolder' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/MarkOfDeath.cs(19,16): warning CS8618: Le propriété 'LogFile' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(73,88): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(73,88): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(81,41): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(83,41): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(150,33): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(150,65): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/SearchFilter.cs(40,48): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(47,30): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(48,30): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(91,30): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(45,16): warning CS8618: Le propriété 'DestDrive' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(144,13): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(155,42): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(293,13): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(327,29): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
  ChiaPlotStatus -> /Users/toto/Chia-Plot-Status/bin/Release/net5.0/ChiaPlotStatus.dll

La génération a réussi.

/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'Language' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'Latest' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'Current' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(33,16): warning CS8618: Le propriété 'current' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'PlotManager' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'ExportJsonCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'ExportYamlCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'ExportCsvCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'AddFolderCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'RemoveFolderCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'IncreaseFontSizeCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'DecreaseFontSizeCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(53,16): warning CS8618: Le propriété 'MarkAsDeadCommand' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/DonationDialog.axaml.cs(26,16): warning CS8618: Le propriété 'Language' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(35,16): warning CS8618: Le propriété 'current' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/DonationDialog.axaml.cs(47,27): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/DonationDialog.axaml.cs(48,56): warning CS8604: Existence possible d'un argument de référence null pour le paramètre 'text' dans 'Task IClipboard.SetTextAsync(string text)'. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le propriété 'TailLineEmitter' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le propriété 'Timer' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le propriété 'Path' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(28,16): warning CS8618: Le champ 'LogLines' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le champ comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(32,16): warning CS8618: Le propriété 'TailLineEmitter' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(32,16): warning CS8618: Le propriété 'Timer' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/LogViewerWindow.axaml.cs(32,16): warning CS8618: Le champ 'LogLines' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le champ comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'BtnClickWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'TextChangeWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'SortChangeWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(26,16): warning CS8618: Le propriété 'ThemeSwitchWorkaround' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(52,27): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(26,25): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(27,25): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(31,25): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(23,16): warning CS8618: Le propriété 'Tmp1Drive' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Statistics/PlottingStatisticsID.cs(23,16): warning CS8618: Le propriété 'Tmp2Drive' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(70,13): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(98,75): warning CS8629: Le type valeur Nullable peut avoir une valeur null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(103,58): warning CS8604: Existence possible d'un argument de référence null pour le paramètre 'arg' dans 'bool Func<string, bool>.Invoke(string arg)'. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/UpdateDialog.axaml.cs(66,20): warning CS8603: Existence possible d'un retour de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(36,23): warning CS8618: Le propriété 'Name' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(37,25): warning CS8618: Le propriété 'Tooltips' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(38,24): warning CS8618: Le propriété 'Columns' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(39,23): warning CS8618: Le propriété 'Fields' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Models/Translation.cs(40,24): warning CS8618: Le propriété 'Buttons' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(91,35): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MainWindow.axaml.cs(92,24): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'Settings' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'Language' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'OnUpdate' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(28,16): warning CS8618: Le propriété 'plotLogReadable' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(43,41): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(58,18): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/Views/MarkOfDeathDialog.axaml.cs(70,17): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(109,51): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/ChiaPlotStatus.cs(67,45): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/MarkOfDeath.cs(19,16): warning CS8618: Le propriété 'LogFolder' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/MarkOfDeath.cs(19,16): warning CS8618: Le propriété 'LogFile' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(73,88): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(73,88): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(81,41): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(83,41): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(150,33): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/Sorter.cs(150,65): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Utils/SearchFilter.cs(40,48): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(47,30): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(48,30): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(91,30): warning CS8601: Existence possible d'une assignation de référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/Logic/Models/PlotLogReadable.cs(45,16): warning CS8618: Le propriété 'DestDrive' non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le propriété comme nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(144,13): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(155,42): warning CS8600: Conversion de littéral ayant une valeur null ou d'une éventuelle valeur null en type non-nullable. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(293,13): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
/Users/toto/Chia-Plot-Status/GUI/ViewModels/MainWindowViewModel.cs(327,29): warning CS8602: Déréférencement d'une éventuelle référence null. [/Users/toto/Chia-Plot-Status/ChiaPlotStatus.csproj]
    71 Avertissement(s)
    0 Erreur(s)

Temps écoulé 00:00:38.20
toto@Mac-mini Chia-Plot-Status % .\bin\Release\net5.0\ChiaPlotStatus
zsh: command not found: .binReleasenet5.0ChiaPlotStatus
`

is the command '.\bin\Release\net5.0\ChiaPlotStatus' is the proper one for Mac?

thanks

--- a/README.md
+++ b/README.md
@@ -118,6 +118,8 @@ windows: `.\ChiaPlotStatusGUI\bin\Release\net5.0\ChiaPlotStatus.exe`
 
 linux: `./ChiaPlotStatusGUI/bin/Release/net5.0/ChiaPlotStatus`
 
+macOS: `dotnet ./ChiaPlotStatusGUI/bin/Release/net5.0/ChiaPlotStatus.dll`
+
 alternatively try `dotnet run --build`.
 
 ## Thanks to

